src/main/java/frc/robot/RobotContainer.java

```diff
@@ -1,29 +1,41 @@
 package frc.robot;
 
-import frc.robot.constants.JsonHandler;
-import frc.robot.constants.fields.HomeField;
-import frc.robot.constants.robots.RobotA;
+import edu.wpi.first.wpilibj.GenericHID;
+import frc.robot.subsystems.swerve.SupplierDriveCMD;
+import frc.robot.subsystems.swerve.Swerve;
 import frc.robot.utilities.DashboardController;
+import frc.robot.utilities.TrigonController;
 
 public class RobotContainer {
-    private RobotA robotConstants;
-    private HomeField fieldConstants;
-    private DashboardController dashboardController;
```

Comment:
why are you mixing 2 diffrent pull requests?

Reply:
too late?

Reply:
Please elaborate